dotcom-rendering/src/components/LatestLinks.importable.tsx

```diff
@@ -0,0 +1,199 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import {
+	type ArticleFormat,
+	timeAgo as timeAgoHasAWeirdInterface,
+} from '@guardian/libs';
+import { palette, space, textSans } from '@guardian/source-foundations';
+import { Hide } from '@guardian/source-react-components';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+import { decidePalette } from '../lib/decidePalette';
+import { revealStyles } from '../lib/revealStyles';
+import { useApi } from '../lib/useApi';
+import type { DCRContainerPalette } from '../types/front';
+import { WithLink } from './CardHeadline';
+
+type Props = {
+	id: string;
+	format: ArticleFormat;
+	direction: 'horizontal' | 'vertical';
+	isDynamo?: true;
+	containerPalette?: DCRContainerPalette;
+};
+
+type Blocks = {
+	blocks: Array<{
+		id: string;
+		title: string;
+		publishedDateTime: number;
+		lastUpdatedDateTime: number;
+		body: string;
+	}>;
+};
+
+const timeAgo = (epoch: number, verbose: boolean) => {
+	const value = timeAgoHasAWeirdInterface(epoch, { verbose });
+
+	if (typeof value === 'string' && value !== '') return value;
+	else return undefined;
+};
+
+const style = css`
+	color: white;
+	display: flex;
+	gap: 5px;
+	padding-bottom: ${space[2]}px;
+`;
+
+const horizontal = css`
+	flex-direction: row;
+`;
+const vertical = css`
+	flex-direction: column;
+	padding: 0 ${space[1]}px;
+`;
+
+const li = css`
+	${textSans.xsmall()}
+	overflow: hidden;
+	flex-grow: 1;
+	height: calc(4 * 1.35em + 2 * ${space[1]}px);
+`;
+
+const dividerStyles = css`
+	border-top: 1px solid currentColor;
+	border-left: 1px solid currentColor;
+`;
+
+const bold = css`
+	${textSans.xsmall({ fontWeight: 'bold' })};
+
+	:before {
+		content: '';
+		height: 0.75em;
+		width: 0.75em;
+		margin-right: ${space[1]}px;
+		display: inline-block;
+		background-color: currentColor;
+		border-radius: 100%;
+	}
+`;
+
+const transparent = css`
+	color: transparent;
+`;
+
+const Time = ({
+	epoch,
+	colour,
+}: {
+	epoch: number;
+	colour: SerializedStyles;
+}) => (
+	<>
+		{' '}
+		<strong css={[bold, colour]}>
```

Comment:
There is a updateTimeElement function in DCR (updateTimeElement.ts) that would keep this time updated if we wrapped it in a <time datetime="..."> element. Not sure whether it's appropriate here but flagging it just in case.